dir.props

```diff
@@ -433,6 +433,12 @@
         <NuGetTargetMoniker>.NETStandard,Version=v1.5</NuGetTargetMoniker>
       </PropertyGroup>
     </When>
+    <When Condition="'$(TargetGroup)'=='netstandard1.6'">
```

Comment:
We don't *need* it yet, because if any project builds in netstandard1.7 then it should be the default as it is the latest. This is the reason why netstandard1.6 wasn't in this list before. But it is a nice to have, so I can add it.